src/Profile/index.js

```diff
@@ -1,5 +1,145 @@
+import { Link } from "react-router-dom";
+import { getMonthName } from "../Utils";
+import "./index.css";
+
 function Profile() {
-    return <h1>Profile</h1>;
-  }
-  export default Profile;
-  
\ No newline at end of file
+  const username = "Brando";
+  const region = "Ryder Hall, MA";
+  const joinDate = new Date(2023, 11, 17);
+
+  const purchasedPokemon = [
+    {
+      purchaseId: 727,
+      timeOfPurchase: new Date(2023, 11, 18, 10, 33),
+      buyerId: 528,
+      sellerId: 491,
+      pokemon: {
+        id: 132,
+        name: "ditto",
+        height: 0.3,
+        weight: 4,
+        types: ["normal"],
+        sprites: {
+          large:
+            "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/132.png",
+          small:
+            "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/132.png",
+        },
+      },
+    },
+  ];
+  const listedPokemon = [
+    {
+      listingId: 900,
+      timeOfListing: new Date(2023, 11, 17, 19, 33),
+      sellerId: 491,
+      pokemon: {
+        id: 132,
+        name: "ditto",
+        height: 0.3,
+        weight: 4,
+        types: ["normal"],
+        sprites: {
+          large:
+            "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/132.png",
+          small:
+            "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/132.png",
+        },
+      },
+    },
+    {
+      timeOfListing: new Date(2023, 11, 17, 19, 34),
+      sellerId: 491,
+      pokemon: {
+        id: 25,
+        name: "pikachu",
+        height: 0.4,
+        weight: 6,
+        types: ["electric"],
+        sprites: {
+          large:
+            "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/25.png",
+          small:
+            "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/25.png",
+        },
+      },
+    },
+  ];
+  const soldPokemon = [
+    {
+      purchaseId: 727,
+      timeOfPurchase: new Date(2023, 11, 18, 10, 33),
+      buyerId: 528,
+      sellerId: 491,
+      pokemon: {
+        id: 132,
+        name: "ditto",
+        height: 0.3,
+        weight: 4,
+        types: ["normal"],
+        sprites: {
+          large:
+            "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/132.png",
+          small:
+            "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/132.png",
+        },
+      },
+    },
+  ];
+
+  return (
+    <div className="row">
+      <div className="col-6 col-sm-6">
+        <img
+          src="https://www.svgrepo.com/show/135058/circle-outline.svg"
+          alt="pfp icon"
+          id="profile-picture"
+        />
```

Comment:
i eventually want to have like, a dozen or so default profile pictures that get automatically assigned to users, like discord's default pfps with the different colors

except they can't change the pfp bc that's work lol
or maybe not idk